Automattic/remote-data-blocks
# Adds Tracks Analytics on backend

### Description

Adds Tracks Analytics by using the Telemetry library available in MU Plugins which will only work if site is hosted on VIP Platform or if the analytics is enabled via filter.

### How to test?

1. Create a site using `vip dev-env` which will have the MU Plugins.
1. Move `remote-data-blocks` plugin to the `plugins` folder.
1. Add filter in `plugin-loader.php` to enable tracking.

    ```php
    add_filter( 'remote_data_blocks_enable_tracks_analytics', '__return_true' );
    ```
1. Comment [these lines](https://github.com/Automattic/remote-data-blocks/blob/78c5cd5a7fdb345b5825932551ca8fc738ac6927/inc/Analytics/TracksAnalytics.php#L32C1-L35C4) to allow tracking on `local` env.
1. Trigger track events, verify that the events are logged on Live View without any error.
